fe/src/main/java/org/apache/doris/common/util/Daemon.java

```diff
@@ -106,8 +106,8 @@ public void run() {
         while (!isStop.get()) {
             try {
                 runOneCycle();
-            } catch (Exception e) {
-                LOG.error("exception: ", e);
+            } catch (Throwable e) {
```

Comment:
Why catch throwable. because derived class such as Error should't be catched. we are only catch exceptions except RuntimeException.

Reply:
If there are Error thrown by Daemon, the daemon thread will be exited while fe is running correctly. No one know this error when thread has been exited.